src/app/student/student.component.html

```diff
@@ -1,3 +1,23 @@
-<p>
-  student works!
-</p>
+<u><h1>Dashboard for ant15</h1></u>
+
+<div class="d"><div class="org first">
+  <a href='certificate1'> <button type="button">View Certificates</button></a>
+</div>
+<div class="cert">
+<b><p>Certificates:5</p></b>
+</div>
+</div>
+<div class="d"><div class="org second">
+   <button type="button">View Certificates</button>
+</div>
+<div class="cert">
+<b><p>Certificates:3</p></b>
+</div>
+</div>
+<div class="d"><div class="org third">
+   <button type="button">View Certificates</button>
+</div>
+<div class="cert">
+<b><p>Certificates:2</p></b>
+</div>
+</div>
```

Comment:
one question, how would I group new certificates that I get.  I'm not able to find the logic for that let me know if I  missed something